homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,511 @@
+"""Constants for the HERE Destination Weather service."""
+from __future__ import annotations
+
+import math
+
+from homeassistant.const import (
+    DEGREE,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_TIMESTAMP,
+    LENGTH_CENTIMETERS,
+    LENGTH_KILOMETERS,
+    PERCENTAGE,
+    PRESSURE_MBAR,
+    SPEED_KILOMETERS_PER_HOUR,
+    TEMP_CELSIUS,
+)
+
+DOMAIN = "here_weather"
+
+DEFAULT_SCAN_INTERVAL = 300
+
+FREEMIUM_REQUESTS_PER_MONTH = 250000
+MAX_UPDATE_RATE_FOR_ONE_CLIENT = math.ceil(
+    (31 * 24 * 3600) / FREEMIUM_REQUESTS_PER_MONTH
+)
+
+CONF_LANGUAGE = "language"
+CONF_OFFSET = "offset"
+CONF_OPTION = "option"
+
+MODE_ASTRONOMY = "forecast_astronomy"
+MODE_HOURLY = "forecast_hourly"
+MODE_DAILY = "forecast_7days"
+MODE_DAILY_SIMPLE = "forecast_7days_simple"
+MODE_OBSERVATION = "observation"
+CONF_MODES = [
+    MODE_ASTRONOMY,
+    MODE_HOURLY,
+    MODE_DAILY,
+    MODE_DAILY_SIMPLE,
+    MODE_OBSERVATION,
+]
+DEFAULT_MODE = MODE_DAILY_SIMPLE
+
+ASTRONOMY_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "sunrise": {"name": "Sunrise", "unit_of_measurement": None, "device_class": None},
+    "sunset": {"name": "Sunset", "unit_of_measurement": None, "device_class": None},
+    "moonrise": {"name": "Moonrise", "unit_of_measurement": None, "device_class": None},
+    "moonset": {"name": "Moonset", "unit_of_measurement": None, "device_class": None},
+    "moonPhase": {
+        "name": "Moon Phase",
+        "unit_of_measurement": "%",
```

Comment:
```suggestion
        "unit_of_measurement": PERCENTAGE,
```

Reply:
Done in https://github.com/home-assistant/core/pull/28910/commits/e1b7683866b634cf21c81d55115f2f650a057c48